gcm/gcm.py

```diff
@@ -39,10 +40,6 @@ class GCMInvalidTtlException(GCMException):
 # Exceptions from Google responses
 
 
-class GCMMissingRegistrationException(GCMException):
```

Comment:
Someone just sent a PR with this exception in #81 which fixes #67 . I think we have to keep the exception to raise it when GCM server returns this error.